contracts/pm/TicketBroker.sol

```diff
@@ -253,8 +278,7 @@ contract TicketBroker {
         return uint256(keccak256(abi.encodePacked(_ticketHash, _recipientRand))) < _winProb;
     }
 
-    function isValidTicketSig(address _sender, bytes _sig, bytes32 _ticketHash) internal pure returns (bool) {
-        address signer = ECDSA.recover(ECDSA.toEthSignedMessageHash(_ticketHash), _sig);
-        return signer != address(0) && _sender == signer;
+    function _isUnlockInProgress(Sender memory sender) internal pure returns (bool) {
```

Comment:
Moved this function here to adhere to a contract layout of state mutating functions -> view functions -> pure functions